eve/main.yml

```diff
@@ -1553,3 +1556,277 @@ stages:
+          env:
+            SOLUTION_VERSION: "%(prop:example_solution_version)s"
+          command: >
+            ssh -F ssh_config bootstrap
+            sudo bash
+            /var/tmp/metalk8s/solutions.sh add-solution --name
+            example-environment --solution example-solution --version
+            "${SOLUTION_VERSION}" --verbose
+          workdir: build/eve/workers/openstack-multiple-nodes/terraform/
+          haltOnFailure: true
+      - ShellCommand:
+          name: Copy example-solution CRs to Bootstrap node
+          env:
+            SSH_CONFIG: >-
+              eve/workers/openstack-multiple-nodes/terraform/ssh_config
+            HOST_CR_PATH: "examples/metalk8s-solution-example/example-operator/deploy/crds/examplesolution_v1alpha1_versionserver_cr.yaml"
+            BOOTSTRAP_CR_PATH: "/tmp/examplesolution_v1alpha1_versionserver_cr.yaml"
+          command: >
+            scp -F $SSH_CONFIG "${HOST_CR_PATH}" bootstrap:"${BOOTSTRAP_CR_PATH}"
+          haltOnFailure: true
+      - ShellCommand:
+          name: Apply example-solution CRs on Bootstrap node
+          command: >
+            ssh -F ssh_config bootstrap
+            sudo kubectl --kubeconfig /etc/kubernetes/admin.conf apply -f
+            /tmp/examplesolution_v1alpha1_versionserver_cr.yaml
+          workdir: build/eve/workers/openstack-multiple-nodes/terraform/
+          haltOnFailure: true
+      - ShellCommand:
+          name: Verify example-solution-operator Deployment
+          env:
+            KUBECONF: "--kubeconfig /etc/kubernetes/admin.conf"
+          command: |
+            ssh -F ssh_config bootstrap --
+            bash -c '
+            ATTEMPTS=0
+            REPLICAS=$(sudo kubectl "$KUBECONF" get deployments example-solution-operator -n example-environment -o custom-columns=:status.availableReplicas)
+            until [ $REPLICAS -eq 1 ]; do
+              $REPLICAS
+              ((ATTEMPTS++))
+              if [ $ATTEMPTS -eq 10 ]; then
+                echo "Could not obtain Replicas after ${ATTEMPTS} attempts" >&2
+                exit 1
+              fi
+              sleep 3
+            done'
+          workdir: build/eve/workers/openstack-multiple-nodes/terraform/
+          haltOnFailure: true
+      - ShellCommand:
+          name: Verify exampleversions Deployment
+          env:
+            KUBECONF: "--kubeconfig /etc/kubernetes/admin.conf"
+          command: |
+            ssh -F ssh_config bootstrap --
+            bash -c '
+            ATTEMPTS=0
+            REPLICAS=$(sudo kubectl "$KUBECONF" get deployments example-versionserver -n example-environment -o custom-columns=:status.availableReplicas)
+            until [ $REPLICAS -eq 3 ]; do
+              $REPLICAS
+              ((ATTEMPTS++))
+              if [ $ATTEMPTS -eq 10 ]; then
+                echo "Could not obtain Replicas after ${ATTEMPTS} attempts" >&2
+                exit 1
+              fi
+              sleep 3
+            done'
```

Comment:
You're not executing the above command here, only executing the result of it.
Should be something like:
```suggestion
            bash -c '
            ATTEMPTS=0
            declare -a GET_REPLICAS=(
                sudo kubectl "$KUBECONFIG" get deployments example-versionserver
                -n example-environment -o custom-columns=:status.availableReplicas
            )
            REPLICAS=$("${GET_REPLICAS[@]}")
            until [ $REPLICAS -eq 3 ]; do
              ((ATTEMPTS++))
              if [ $ATTEMPTS -eq 10 ]; then
                echo "Could not obtain Replicas after ${ATTEMPTS} attempts" >&2
                exit 1
              fi
              sleep 3
              REPLICAS=$("${GET_REPLICAS[@]}")
            done'
```
Or even:
```suggestion
            bash -c '
            declare -a GET_REPLICAS=(
                sudo kubectl "$KUBECONFIG" get deployments example-versionserver
                -n example-environment -o custom-columns=:status.availableReplicas
            )
            for ((attempts=0, replicas=$("${GET_REPLICAS[@]}"); replicas != 3; replicas=$("${GET_REPLICAS[@]}"), ++attempts)); do
              if [ $attempts -eq 10 ]; then
                echo "Could not obtain Replicas after $attempts attempts" >&2
                exit 1
              fi
              sleep 3
            done'
```